src/tests/Myr.test.js

```diff
@@ -40,6 +40,13 @@ describe("Updates to Myr's Model", () => {
         expect(color).not.toBeUndefined();
         expect(colorRegEx.test(color)).toBeTruthy();
     });
+
+    it("pick Random Color out of a list of colors", () => {
```

Comment:
This test ensures that a color was set, but not that it was pulled from the array. Can we modify this test to make sure it's from the array?

Reply:
`expect(colors.includes(color)).toBeTruthy();`
this line should do that, i think